tailwind.config.js

```diff
@@ -34,9 +35,39 @@ module.exports = {
       colors: {
         "blue-dark": "#001342",
         "green-light": "#8ACC50",
-        "green-lighter": "#116355",
+        "green-darker": "#116355",
+        "gray-dark": "rgba(4, 4, 4, 0.51)",
+        "gray-light": "#F1F4F6",
+        "blue-dark": "#001342",
+      },
+
+      height: {
+        line: "0.2rem",
+        "maiborn-video": "39rem",
```

Comment:
Avoid this way of naming height, instead do this
```suggestion
        156: "39rem",
```
For a proper naming convention and sizing, you can use this simple formula: just multiply your custom rem value by 0.25, and you'll get the right variable name.